mars/services/task/execution/ray/executor.py

```diff
@@ -296,10 +337,15 @@ async def execute_subtask_graph(
                 list(key_to_input.keys()),
                 *key_to_input.values(),
             )
+            self._cur_stage_unfinished_subtasks.add(subtask)
+            self._cur_stage_submitted_subtasks.add(subtask)
             if output_count == 0:
                 continue
             elif output_count == 1:
                 output_object_refs = [output_object_refs]
+            self._cur_stage_subtask_to_output_object_refs[subtask] = set(
+                output_object_refs
```

Comment:
Do we need to track all the output object refs of the subtask? May be one output object ref is enough.

Reply:
Fixed. Thanks.